rails/README.md

```diff
@@ -112,6 +112,13 @@ others' Rails work, look in particular for:
   is a related todo in the project management system to add it during the
   staging and production deploys.
 
+## Asset Management
+
+- Use [ActiveStorage] to manage file uploads that live on ActiveRecord objects.
+- Avoid using live storage backends like S3 or Azure in tests.
```

Comment:
I could be swayed to use the stronger "Don't" language here but left it slightly more permissive given I'd trust if we did "need" to do this, there'd be reasons™.

Reply:
I don't think is that strong since one aspect that we also promote about writing tests is that they should not use the internet

Reply:
I'm down for the stronger "Don't" 👍